packages/web-api/src/cron/cronService.ts

```diff
@@ -62,11 +63,49 @@ export class CronService {
         timezone: 'Australia/Sydney'
       }
     );
-
     this.tasks.push(task);
     task.start();
+    console.debug('Scheduled data specification update job (daily at 2:00 AM AEDT)');
+  }
+
+  /**
+   * Schedule the job timeout cleanup
+   * Runs every hour to clean up stale jobs
+   */
+  private scheduleJobTimeoutCleanup(): void {
+    console.debug('Scheduling job timeout cleanup...');
+    const task = schedule(
+      // Every hour at minute 0
+      '0 * * * *',
+      async () => {
+        try {
+          console.debug('Starting scheduled job timeout cleanup...');
+          const result = await timeoutJobs({
+            timeoutThresholdMinutes: config.jobExpiryMinutes
+          });
 
-    console.debug('Scheduled data specification update job (daily at 2:00 AM UTC)');
+          if (result.totalTimedOut > 0) {
+            console.debug(`Timed out ${result.totalTimedOut} job(s):`);
+            Object.entries(result.byType).forEach(([type, count]) => {
+              console.debug(`  ${type}: ${count}`);
+            });
+            console.debug(`  Job IDs: ${result.timedOutJobIds.join(', ')}`);
+          } else {
+            console.debug('No jobs needed to be timed out');
+          }
+        } catch (error) {
+          console.error('Failed to run scheduled job timeout cleanup:', error);
+        }
+      },
+      {
+        timezone: 'Australia/Sydney'
```

Comment:
thinking because of DST better to set an offset that never changes rather than this timezone, which will change offsets with DST